blog/en/multimodal-semantic-search-with-images-and-text.md

```diff
@@ -11,11 +11,7 @@ recommend: true
 canonicalUrl: https://milvus.io/blog/multimodal-semantic-search-with-images-and-text.md
 ---
 
-
-
-
-
-<iframe width=“100%” height="315" src="https://www.youtube.com/embed/bxE0_QYX_sU?si=PkOHFcZto-rda1Fv" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/bxE0_QYX_sU?si=PkOHFcZto-rda1Fv" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I think for this to work we have to remove `height="315"` but keep `width="100%"`